projects/packages/scheduled-updates/src/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-update-schedules.php

```diff
@@ -179,13 +179,21 @@ public function create_item( $request ) {
 		$plugins  = $request['plugins'];
 		usort( $plugins, 'strnatcasecmp' );
 
-		$event = Scheduled_Updates::create_scheduled_update( $schedule['timestamp'], $schedule['interval'], $plugins );
+		$event = wp_schedule_event( $schedule['timestamp'], $schedule['interval'], Scheduled_Updates::PLUGIN_CRON_HOOK, $plugins, true );
 
 		if ( is_wp_error( $event ) ) {
+			// If the schedule could not be created, return an error.
+			$event->add_data( array( 'status' => 500 ) );
```

Comment:
Should this be 404 too?

Reply:
I set it to 404 for completeness. Thanks.